apis/core.oam.dev/v1alpha1/policy_types.go

```diff
@@ -0,0 +1,36 @@
+/*
+Copyright 2021 The KubeVela Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+const (
+	// TopologyPolicyType refers to the type of topology policy
+	TopologyPolicyType = "topology"
+	// OverridePolicyType refers to the type of override policy
+	OverridePolicyType = "override"
+)
+
+// TopologyPolicySpec defines the spec of topology policy
+type TopologyPolicySpec struct {
+	Clusters        []string          `json:"clusters,omitempty"`
+	ClusterSelector map[string]string `json:"clusterSelector,omitempty"`
```

Comment:
the name of selector also looks obscure to me, is it the label selecting or the claim selecting? the latter is accepted by sig-mc as a standard

Reply:
It is label selecting, which is a direct way for user to use. One of the lack here is we still do not have convenient `vela cluster label` command to help user manage cluster labels.